### PR TITLE
Add Google DeepVariant as a germline variant caller

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
This PR adds support for Google DeepVariant as a germline variant caller. It can be run on its own or alongside other variant callers. Note that the default model is WES, which needs to be change for WGS data. 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [x] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [x] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -62,6 +62,8 @@ params {
   umi = null // no umi
   read_structure1 = null // no umi
   read_structure2 = null // no umi
+  model_type = 'WGS'  // For DeepVariant
```
Comment (thomasyu888) [inline nextflow.config:65]: These parameters are specific to deep variant? Is there a nextflow convention to check for parameter inputs?

**Review by thomasyu888 — APPROVED** [review]
LGTM, could you by chance screenshot a successful run using deep variant? If not, that's ok.

File: nextflow.config (hunk 1 above)
Comment (BrunoGrandePhD) [reply nextflow.config:65]: Yeah, nf-core has JSON schemas with each pipeline. I've updated the one for Sarek to account for the new parameter.

**Comment by BrunoGrandePhD** [comment]
Here's one run that was successful: 

image

**Comment by thomasyu888** [comment]
👍

**Pull request merged**